pkg/odo/cli/init/init.go

```diff
@@ -248,7 +248,7 @@ func NewCmdInit(name, fullName string) *cobra.Command {
 	}
 	clientset.Add(initCmd, clientset.PREFERENCE, clientset.FILESYSTEM, clientset.REGISTRY, clientset.INIT)
 
-	initCmd.Flags().String(backend.FLAG_NAME, "", "name of the component to create")
+	initCmd.Flags().String(backend.FLAG_NAME, "", "name of the component to create; it should follow the RFC 1123 Label Names")
```

Comment:
Same comment about us requiring non-all-numeric names:

```suggestion
	initCmd.Flags().String(backend.FLAG_NAME, "", "name of the component to create; it must follow the RFC 1123 Label Name standard and not be all-numeric")
```

Reply:
The suggested change seems to have disappeared..